Contents/Code/__init__.py

```diff
@@ -100,18 +112,35 @@ def Update(self, metadata, media, lang, force, movie):
         flags = flags | Prefs['hideUsefulMiscTags'] << 3 #0b01000 : Hide Useful Miscellaneous Tags
         flags = flags | Prefs['hideSpoilerTags']    << 4 #0b10000 : Hide Plot Spoiler Tags
 
+        if movie:
+            series = HttpReq("api/serie/fromep?id=%s&level=3&allpics=1&tagfilter=%d" % (aid, flags))
+            movie_episode_data = HttpReq("api/ep?id=%s" % (aid))
+
```

Comment:
Blank line not needed, group 2ith other variable assignments

Reply:
I didn't understand, I kept them separate because they were API calls.
Should I remove line 118?